Dockerfile

```diff
@@ -3,7 +3,7 @@ MAINTAINER Jason Wilder <jason@influxdb.com>
 
 ENV VERSION 0.4.0
 ENV DOWNLOAD_URL https://github.com/jwilder/docker-gen/releases/download/$VERSION/docker-gen-linux-amd64-$VERSION.tar.gz
-ENV DOCKER_HOST unix:///tmp/docker.sock
+ENV DOCKER_HOST unix:///var/run/docker.sock
```

Comment:
Isn't this going to break everyone who currently invokes their `docker-gen` container with `-v /var/run/docker.sock:/tmp/docker.sock`?